### DISCUSSION TITLE
NTR Coordination of growth and division

### DISCUSSION THREAD

**Discussion by suzialeksander** [geneontology/helpdesk#408] [Ideas]
I'm at a conference and there has been a request for a term/branch relating to "coordination of growth and division". 

I'll follow up to see if there's a reference, but for now just making a note and checking if there is any history with this kind of a category.  @pgaudet and @vanaukenk does this topic sound familiar?

**Comment by pgaudet** [comment]
Would 'GO:0031568 mitotic G1 cell size control checkpoint signaling' work?
